_blog.yml

```
@@ -1570,3 +1570,13 @@
   date: Dec 15, 2022
   tags:
     - ethics
+
+- local: share-vision-ds
```

```suggestion
- local: sharing-vision-datasets
```

make `ds` more explicit